test/JDBC/expected/babel_726-vu-verify.out

```diff
@@ -0,0 +1,575 @@
+GO
+~~START~~
+image
+A678F151C7532C479BE11C08942342D7
+~~END~~
+
+
+-- uniqueidentifier and nvarchar
+select coalesce(a21, a22) from babel_726_t1
+GO
+~~START~~
+uniqueidentifier
+51F178A6-53C7-472C-9BE1-1C08942342D7
+~~END~~
+
+select coalesce(a22, a21) from babel_726_t1
+GO
+~~START~~
+uniqueidentifier
+~~ERROR (Code: 33557097)~~
+
+~~ERROR (Message: invalid input syntax for type uuid: "asdfgh")~~
+
+
+-- nvarchar and nchar
+select coalesce(a22, a23) from babel_726_t1
+GO
+~~START~~
+nvarchar
+asdfgh
+~~END~~
+
+select coalesce(a23, a22) from babel_726_t1
+GO
+~~START~~
+nvarchar
+a              
+~~END~~
+
+
+-- nchar and varchar
+select coalesce(a23, a24) from babel_726_t1
+GO
+~~ERROR (Code: 33557097)~~
+
+~~ERROR (Message: The string size for the given CHAR/NCHAR data is not defined. Please use an explicit CAST or CONVERT to CHAR(n)/NCHAR(n))~~
+
+select coalesce(a24, a23) from babel_726_t1
+GO
+~~ERROR (Code: 33557097)~~
+
+~~ERROR (Message: The string size for the given CHAR/NCHAR data is not defined. Please use an explicit CAST or CONVERT to CHAR(n)/NCHAR(n))~~
+
+
+-- varchar and char
+select coalesce(a24, a25) from babel_726_t1
+GO
+~~START~~
+varchar
+asdfg
+~~END~~
+
+select coalesce(a25, a24) from babel_726_t1
+GO
+~~START~~
+varchar
+b              
+~~END~~
+
+
+-- char and varbinary
+select coalesce(a25, a26) from babel_726_t1
+GO
+~~ERROR (Code: 33557097)~~
+
+~~ERROR (Message: COALESCE could not convert type varbinary to bpchar)~~
+
+select coalesce(a26, a25) from babel_726_t1
+GO
+~~ERROR (Code: 33557097)~~
+
+~~ERROR (Message: COALESCE could not convert type varbinary to bpchar)~~
+
+
+-- varbinary and binary
+select coalesce(a26, a27) from babel_726_t1
+GO
+~~START~~
+varbinary
+000004D2
+~~END~~
+
+select coalesce(a27, a26) from babel_726_t1
+GO
+~~START~~
+varbinary
+01
+~~END~~
+
+
+-- numeric and varchar
+select coalesce(a28, a24) from babel_726_t1
+GO
+~~START~~
+numeric
+235.00000000
+~~END~~
+
+select coalesce(a24, a28) from babel_726_t1
+GO
+~~START~~
+numeric
+~~ERROR (Code: 33557097)~~
+
+~~ERROR (Message: invalid input syntax for type numeric: "asdfg")~~
+
+
+-- numeric and float
+select coalesce(a28, a8) from babel_726_t1
+GO
+~~START~~
+float
+235.0
+~~END~~
+
+select coalesce(a8, a28) from babel_726_t1
+GO
+~~START~~
+float
+1.05
+~~END~~
+
+
+EXEC sp_babelfish_configure 'babelfishpg_tsql.escape_hatch_rowversion', 'strict';
+go
+
+select coalesce(a,b,c) from babel_726_t2
+go
+~~START~~
+float
+1.0
+2.01
+1.02
+~~END~~
+
+
+insert into babel_726_t2 values (NULL, 'abcde', 1.02)
+go
+~~ROW COUNT: 1~~
+
+
+select coalesce(a,b,c) from babel_726_t2
+go
+~~START~~
+float
+1.0
+2.01
+1.02
+~~ERROR (Code: 33557097)~~
+
+~~ERROR (Message: invalid input syntax for type double precision: "abcde")~~
+
+
+DECLARE @ID_UNDE INT = 7;
+select COALESCE(@ID_UNDE, 'Não Informado')
+go
+~~START~~
+int
+7
+~~END~~
+
+
+exec babel_726_vu_prepare_p1
+go
+~~START~~
+nvarchar
+x
+~~END~~
+
+
+select babel_726_vu_prepare_f1()
```

Comment:
So this means that if this procedure was executed on prior versions, it would give an error but starting this version, we will get the correct output?